test/Microsoft.TestPlatform.CommunicationUtilities.UnitTests/TestRequestSenderTests.cs

```diff
@@ -156,7 +156,7 @@ public void EndSessionShouldNotSendSessionEndMessageIfTestHostProcessExited()
 
             this.testRequestSender.EndSession();
 
-            this.mockDataSerializer.Verify(ds => ds.SerializeMessage(MessageType.SessionEnd), Times.Once);
+            this.mockDataSerializer.Verify(ds => ds.SerializeMessage(MessageType.SessionEnd), Times.Never);
```

Comment:
Yes. This is correcting the test. Since we were not setting IsOperationComplete in OnClientProcessExit, this test was passing. The test should not send end message if client process exited